app/db/models/user.js

```diff
@@ -204,11 +204,26 @@ module.exports = (Sequelize, db) => {
     });
   };
 
+  // Verifies user's email by setting their account type to STANDARD if it is RESTRICTED
+  User.prototype.validateEmail = function () {
+    if (this.accessType === 'RESTRICTED') {
+      return this.update({ accessType: 'STANDARD' });
+    }
+    return new Promise((resolve) => {
+      resolve();
+    });
+  };
+
   User.getLeaderboard = function (offset, limit) {
     if (!offset || offset < 0) offset = 0;
     if (!limit || limit < 0) limit = undefined;
     return this.findAll({
-      where: { accessType: { [Sequelize.Op.not]: 'ADMIN' } },
+      where: {
+        [Sequelize.Op.and]: [
+          { accessType: { [Sequelize.Op.not]: 'ADMIN' } },
+          { accessType: { [Sequelize.Op.not]: 'RESTRICTED' } },
```

Comment:
update to `state ACTIVE`

Reply:
now blocking for `accessType ADMIN`, `state PENDING` and `state BLOCKED`